pages.nl/common/[[.md

```diff
@@ -0,0 +1,37 @@
+# [[
+
+> Controleer bestandstypen en vergelijk waarden.
+> Retourneert 0 als de voorwaarde resulteert in waar, 1 als deze resulteert in onwaar.
```

Comment:
```suggestion
> Retourneert een status van 0 als de voorwaarde resulteert in waar, 1 als deze resulteert in onwaar.
```
From Seth's PR at #10205. Feel free to change the above suggestion (as I used Google Translate for it).

Reply:
Looks like Google did it just fine!